tests/Feature/Modules/Locations/RentalConditionReportTest.php

```
@@ -0,0 +1,429 @@
+});
+
+it('logs and ignores failures when attaching a photo', function () {
+    Log::spy();
+    $report = Mockery::mock(RentalConditionReport::class)->makePartial();
+    $report->shouldReceive('addMediaFromBase64')->once()->andThrow(new Exception('invalid image'));
+
+    $service = app(RentalConditionReportService::class);
+    $result = $service->attachPhoto($report, validImageBase64());
+
+    expect($result)->toBeFalse();
+    Log::shouldHaveReceived('error')->once();
```

PHPStan (2.2.7)

[error] Call to an undefined static method Illuminate\Support\Facades\Log::shouldHaveReceived().


(staticMethod.notFound)